dim-core/src/routes/settings.rs

```diff
@@ -61,6 +66,7 @@ impl Default for GlobalSettings {
             verbose: false,
             secret_key: None,
             enable_hwaccel: true,
+            version: version.into(),
```

Comment:
@vgarleanu cautioned me about how this could be overridden;

> just remember to annotate globalsettings with serde(skip) otherwise the version info could get overrided by the config.toml which would get super confusing

I realized that I don't actually know how to implement this properly as annotating `#[serde(skip)]` for version will just exclude the version from the response.

Reply:
mhm yeah youre right, i think the only way to solve this is to have a interim GlobalSettings thats used for the web response and one for the serialization to the disk

Reply:
Ok, this is addressed in https://github.com/Dusk-Labs/dim/pull/580/commits/7f821173f6d30f51577da93603fa5b3c9da63737, but I'll rebase this after https://github.com/Dusk-Labs/dim/pull/578 is merged since there are route conflicts now that aren't worth resolving twice.